To: vim_dev@googlegroups.com
Subject: Patch 7.3.298
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.298
Problem:    Built-in colors are different from rgb.txt.
Solution:   Adjust the color values. (Benjamin Haskell)
Files:	    src/gui_photon.c, src/gui_w48.c


*** ../vim-7.3.297/src/gui_photon.c	2011-08-10 12:19:00.000000000 +0200
--- src/gui_photon.c	2011-09-07 18:55:09.000000000 +0200
***************
*** 2007,2014 ****
      static GuiColourTable table[] =
      {
  	{"Black",	    RGB(0x00, 0x00, 0x00)},
! 	{"DarkGray",	    RGB(0x80, 0x80, 0x80)},
! 	{"DarkGrey",	    RGB(0x80, 0x80, 0x80)},
  	{"Gray",	    RGB(0xC0, 0xC0, 0xC0)},
  	{"Grey",	    RGB(0xC0, 0xC0, 0xC0)},
  	{"LightGray",	    RGB(0xD3, 0xD3, 0xD3)},
--- 2007,2014 ----
      static GuiColourTable table[] =
      {
  	{"Black",	    RGB(0x00, 0x00, 0x00)},
! 	{"DarkGray",	    RGB(0xA9, 0xA9, 0xA9)},
! 	{"DarkGrey",	    RGB(0xA9, 0xA9, 0xA9)},
  	{"Gray",	    RGB(0xC0, 0xC0, 0xC0)},
  	{"Grey",	    RGB(0xC0, 0xC0, 0xC0)},
  	{"LightGray",	    RGB(0xD3, 0xD3, 0xD3)},
***************
*** 2037,2055 ****
  	{"LightRed",	    RGB(0xFF, 0xA0, 0xA0)},
  	{"DarkBlue",	    RGB(0x00, 0x00, 0x80)},
  	{"Blue",	    RGB(0x00, 0x00, 0xFF)},
! 	{"LightBlue",	    RGB(0xA0, 0xA0, 0xFF)},
  	{"DarkGreen",	    RGB(0x00, 0x80, 0x00)},
  	{"Green",	    RGB(0x00, 0xFF, 0x00)},
! 	{"LightGreen",	    RGB(0xA0, 0xFF, 0xA0)},
  	{"DarkCyan",	    RGB(0x00, 0x80, 0x80)},
  	{"Cyan",	    RGB(0x00, 0xFF, 0xFF)},
! 	{"LightCyan",	    RGB(0xA0, 0xFF, 0xFF)},
  	{"DarkMagenta",	    RGB(0x80, 0x00, 0x80)},
  	{"Magenta",	    RGB(0xFF, 0x00, 0xFF)},
  	{"LightMagenta",    RGB(0xFF, 0xA0, 0xFF)},
  	{"Brown",	    RGB(0x80, 0x40, 0x40)},
  	{"Yellow",	    RGB(0xFF, 0xFF, 0x00)},
! 	{"LightYellow",	    RGB(0xFF, 0xFF, 0xA0)},
  	{"SeaGreen",	    RGB(0x2E, 0x8B, 0x57)},
  	{"Orange",	    RGB(0xFF, 0xA5, 0x00)},
  	{"Purple",	    RGB(0xA0, 0x20, 0xF0)},
--- 2037,2055 ----
  	{"LightRed",	    RGB(0xFF, 0xA0, 0xA0)},
  	{"DarkBlue",	    RGB(0x00, 0x00, 0x80)},
  	{"Blue",	    RGB(0x00, 0x00, 0xFF)},
! 	{"LightBlue",	    RGB(0xAD, 0xD8, 0xE6)},
  	{"DarkGreen",	    RGB(0x00, 0x80, 0x00)},
  	{"Green",	    RGB(0x00, 0xFF, 0x00)},
! 	{"LightGreen",	    RGB(0x90, 0xEE, 0x90)},
  	{"DarkCyan",	    RGB(0x00, 0x80, 0x80)},
  	{"Cyan",	    RGB(0x00, 0xFF, 0xFF)},
! 	{"LightCyan",	    RGB(0xE0, 0xFF, 0xFF)},
  	{"DarkMagenta",	    RGB(0x80, 0x00, 0x80)},
  	{"Magenta",	    RGB(0xFF, 0x00, 0xFF)},
  	{"LightMagenta",    RGB(0xFF, 0xA0, 0xFF)},
  	{"Brown",	    RGB(0x80, 0x40, 0x40)},
  	{"Yellow",	    RGB(0xFF, 0xFF, 0x00)},
! 	{"LightYellow",	    RGB(0xFF, 0xFF, 0xE0)},
  	{"SeaGreen",	    RGB(0x2E, 0x8B, 0x57)},
  	{"Orange",	    RGB(0xFF, 0xA5, 0x00)},
  	{"Purple",	    RGB(0xA0, 0x20, 0xF0)},
*** ../vim-7.3.297/src/gui_w48.c	2011-08-10 17:07:56.000000000 +0200
--- src/gui_w48.c	2011-09-07 18:55:09.000000000 +0200
***************
*** 1411,1422 ****
      static guicolor_tTable table[] =
      {
  	{"Black",		RGB(0x00, 0x00, 0x00)},
! 	{"DarkGray",		RGB(0x80, 0x80, 0x80)},
! 	{"DarkGrey",		RGB(0x80, 0x80, 0x80)},
  	{"Gray",		RGB(0xC0, 0xC0, 0xC0)},
  	{"Grey",		RGB(0xC0, 0xC0, 0xC0)},
! 	{"LightGray",		RGB(0xE0, 0xE0, 0xE0)},
! 	{"LightGrey",		RGB(0xE0, 0xE0, 0xE0)},
  	{"Gray10",		RGB(0x1A, 0x1A, 0x1A)},
  	{"Grey10",		RGB(0x1A, 0x1A, 0x1A)},
  	{"Gray20",		RGB(0x33, 0x33, 0x33)},
--- 1411,1422 ----
      static guicolor_tTable table[] =
      {
  	{"Black",		RGB(0x00, 0x00, 0x00)},
! 	{"DarkGray",		RGB(0xA9, 0xA9, 0xA9)},
! 	{"DarkGrey",		RGB(0xA9, 0xA9, 0xA9)},
  	{"Gray",		RGB(0xC0, 0xC0, 0xC0)},
  	{"Grey",		RGB(0xC0, 0xC0, 0xC0)},
! 	{"LightGray",		RGB(0xD3, 0xD3, 0xD3)},
! 	{"LightGrey",		RGB(0xD3, 0xD3, 0xD3)},
  	{"Gray10",		RGB(0x1A, 0x1A, 0x1A)},
  	{"Grey10",		RGB(0x1A, 0x1A, 0x1A)},
  	{"Gray20",		RGB(0x33, 0x33, 0x33)},
***************
*** 1441,1459 ****
  	{"LightRed",		RGB(0xFF, 0xA0, 0xA0)},
  	{"DarkBlue",		RGB(0x00, 0x00, 0x80)},
  	{"Blue",		RGB(0x00, 0x00, 0xFF)},
! 	{"LightBlue",		RGB(0xA0, 0xA0, 0xFF)},
  	{"DarkGreen",		RGB(0x00, 0x80, 0x00)},
  	{"Green",		RGB(0x00, 0xFF, 0x00)},
! 	{"LightGreen",		RGB(0xA0, 0xFF, 0xA0)},
  	{"DarkCyan",		RGB(0x00, 0x80, 0x80)},
  	{"Cyan",		RGB(0x00, 0xFF, 0xFF)},
! 	{"LightCyan",		RGB(0xA0, 0xFF, 0xFF)},
  	{"DarkMagenta",		RGB(0x80, 0x00, 0x80)},
  	{"Magenta",		RGB(0xFF, 0x00, 0xFF)},
  	{"LightMagenta",	RGB(0xFF, 0xA0, 0xFF)},
  	{"Brown",		RGB(0x80, 0x40, 0x40)},
  	{"Yellow",		RGB(0xFF, 0xFF, 0x00)},
! 	{"LightYellow",		RGB(0xFF, 0xFF, 0xA0)},
  	{"DarkYellow",		RGB(0xBB, 0xBB, 0x00)},
  	{"SeaGreen",		RGB(0x2E, 0x8B, 0x57)},
  	{"Orange",		RGB(0xFF, 0xA5, 0x00)},
--- 1441,1459 ----
  	{"LightRed",		RGB(0xFF, 0xA0, 0xA0)},
  	{"DarkBlue",		RGB(0x00, 0x00, 0x80)},
  	{"Blue",		RGB(0x00, 0x00, 0xFF)},
! 	{"LightBlue",		RGB(0xAD, 0xD8, 0xE6)},
  	{"DarkGreen",		RGB(0x00, 0x80, 0x00)},
  	{"Green",		RGB(0x00, 0xFF, 0x00)},
! 	{"LightGreen",		RGB(0x90, 0xEE, 0x90)},
  	{"DarkCyan",		RGB(0x00, 0x80, 0x80)},
  	{"Cyan",		RGB(0x00, 0xFF, 0xFF)},
! 	{"LightCyan",		RGB(0xE0, 0xFF, 0xFF)},
  	{"DarkMagenta",		RGB(0x80, 0x00, 0x80)},
  	{"Magenta",		RGB(0xFF, 0x00, 0xFF)},
  	{"LightMagenta",	RGB(0xFF, 0xA0, 0xFF)},
  	{"Brown",		RGB(0x80, 0x40, 0x40)},
  	{"Yellow",		RGB(0xFF, 0xFF, 0x00)},
! 	{"LightYellow",		RGB(0xFF, 0xFF, 0xE0)},
  	{"DarkYellow",		RGB(0xBB, 0xBB, 0x00)},
  	{"SeaGreen",		RGB(0x2E, 0x8B, 0x57)},
  	{"Orange",		RGB(0xFF, 0xA5, 0x00)},
*** ../vim-7.3.297/src/version.c	2011-09-07 18:47:19.000000000 +0200
--- src/version.c	2011-09-07 18:55:27.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     298,
  /**/

-- 
You can test a person's importance in the organization by asking how much RAM
his computer has.  Anybody who knows the answer to that question is not a
decision-maker.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
